plugins/source/oracle/docs/overview.md

```diff
@@ -25,10 +25,26 @@ The following example sets up the Oracle plugin, and connects it to a postgresql
 
 See [tables](/docs/plugins/sources/oracle/tables) for a full list of available tables.
 
-## Oracle Spec
+### Oracle Spec
 
 This is the (nested) spec used by Oracle Source Plugin
 
-- `concurrency` (int, optional, default: 10000):
+- `concurrency` (`integer`) (optional) (default: `10000`)
+
   The best effort maximum number of Go routines to use.
   Lower this number to reduce memory usage.
+
+## Dedicated regions
+
+[OCI Dedicated regions](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/dedicatedregions.htm) can be accessed via the following procedure:
+
+1. Specify the region to be used for the discovery as the dedicated region.
+   The following options are available:
+   * Set `OCI_CLI_region` environment variable
+   * Set `TF_VAR_region` environment variable
```

Comment:
nit: I would probably leave the `TF_VAR` option out as it's really only there for backwards-compatibility

Reply:
01b45bc6ba18b8ebded805e32abb02204213abe2